scripts/db_migrator.py

```diff
@@ -51,6 +51,34 @@ def migrate_pfc_wd_table(self):
         self.configDB.delete_table('PFC_WD_TABLE')
 
 
+    def migrate_interface_table(self):
+        # Migrate all data entries for Interface table
```

Comment:
can we get better description here? what do you mean by migrate all data entries fro interface table? use """ """ for comments.